jetty-server/src/main/config/etc/jetty-secure-redirect.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Get id="Handler" name="handler"/>
+  <Call name="setHandler">
+    <Arg>
+      <New class="org.eclipse.jetty.server.handler.HandlerList">
```

Comment:
`SecureRedirectHandler` is a handler wrapper, so you should not need this new HandlerList.
You can just do a `Server.insertHandler` to inject a secure redirect handler between the server and whatever its original handler was.